protocol/protocol.md

```diff
@@ -69,105 +77,230 @@ Syntax:
+
 #### Properties
 
-The etag value must be "quoted". 
+In the response body the value of resourcetype property MUST NOT contain whitespaces (implementation as of owncloud client 1.5):
 
-In the response body the value of resourcetype property MUST NOT contain whitespaces (implementation as of owncloud client 1.5):   
-   
         <d:resourcetype><d:collection/></d:resourcetype>
 
 #### Headers
 
 PROPFIND Depth:infinity is not supported: client will try this for compatibility with older code  but server may choose to return 501 (not implemented).
 
+### Quota Check Call
 
-
-### Quota check
+Client displays user quota information. For that it does a PROPFIND call to the root of the 
+user space on the webdav server.
 
 Syntax:
-	    
-	> PROPFIND /remote.php/webdav/ HTTP/1.1
+
+	PROPFIND /remote.php/webdav/ HTTP/1.1
 	Depth: 0
-	
+
 	<?xml version="1.0" ?>
 	  <d:propfind xmlns:d="DAV:"><d:prop>
-	      <d:quota-available-bytes/>   
-	      <d:quota-used-bytes/>  
+	      <d:quota-available-bytes/>
+	      <d:quota-used-bytes/>
 	  </d:prop></d:propfind>
 
 Response: 207
 
 Reponse body example:
 
-    < PROPFIND
-    
-    <?xml version="1.0" encoding="utf-8"?>
-     <d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
+    <?xml version='1.0' encoding='utf-8'?>
+    <d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
       <d:response>
-        <d:href>/remote.php/webdav/</d:href>
+        <d:href>/oc/remote.php/webdav/</d:href>
         <d:propstat>
           <d:prop>
-            <d:quota-available-bytes>107347967021</d:quota-available-bytes>
-            <d:quota-used-bytes>26215379</d:quota-used-bytes>
+            <oc:id>00000003ocobzus5kn6s</oc:id>
+            <oc:permissions>RDNVCK</oc:permissions>
+            <oc:size>592610518</oc:size>
+            <d:quota-available-bytes>14883876864</d:quota-available-bytes>
+            <d:quota-used-bytes>592610518</d:quota-used-bytes>
           </d:prop>
           <d:status>HTTP/1.1 200 OK</d:status>
         </d:propstat>
       </d:response>
-     </d:multistatus>
+    </d:multistatus>
+    
+Besides the actual quota sizes in bytes the server also returns the fileId and the permissions
+of the top directory.
 
-### Stat top-level directory
+### Connection Validation Call
+
+To detect if there is still a vital uplink to the server, client does a PROPFIND request
+to the top level directory to query the last modified timestamp. 
+
+This call needs working authentication and is used to verify that authentication and the
+network connection in general is still up and running. Depending on the result of this call
+client goes to offline mode or opens the authentication dialog to ask for new password.
 
 Syntax:
 
-    > PROPFIND /remote.php/webdav/ HTTP/1.1
+    PROPFIND /remote.php/webdav/ HTTP/1.1
```

Comment:
I will add this little character ">" here because it allows to automatically validate XML payload describe in this document.